<launch>
	<node pkg="vicon_mocap" type="vicon_recv_direct" name="vicon_recv_direct">
		<param name="stream_mode" value="ClientPullPreFetch" type="str" />
		<param name="datastream_hostport" value="192.168.1.2:801" type="str" />
		<param name="subject_name" value="pelican1" type="str" />
		<param name="segment_name" value="pelican1" type="str" />
		<param name="tf_ref_frame_id" value="/enu" type="str" />
		<param name="tf_tracked_frame_id" value="/pelican1/flyer_vicon" type="str" />
		<param name="update_rate" value="100" type="double" />
        	<param name="vicon_capture_rate" value="100" type="double" />
		</node>
</launch>
